.gitignore

```diff
@@ -32,3 +32,6 @@ node_modules2
 /test/package/**/yarn.lock
 /test/package/**/dist/
 scripts/dockolith
+
+.yarnrc
+.npmrc
```

Comment:
new?

Reply:
my local need this to make sure public registry is used

Reply:
ack. it's not going to cause issues for others?